### DISCUSSION TITLE
Artillery Cloud stops reporting metrics if local terminal session that triggered a Fargate run loses connection or is closed.

### DISCUSSION THREAD

**Discussion by DanSmithSC** [artilleryio/artillery#3295] [Artillery]
While doing some Artillery Cloud runs that were triggering Fargate tests, I've noticed that if the Local terminal session which triggered the run loses connection or for example is closed It seems that Artillery stops reporting metrics on the run. 

The Artillery Workers will continue to run and generate load, but no metrics will be further captured in the Artillery Cloud Dashboard.

Is this a known behavior? - Even if it's running from within Fargate why is the local terminal session required to record metrics up to the Cloud?

**Comment by bernardobridge** [comment]
Hello @DanSmithSC 👋 ,

The metrics (and other information) are sent to Artillery Cloud from the main terminal session that triggered the test (as that is where metrics are aggregated). So yes, currently that terminal session would need to remain while the test runs.
